provider/nbd_image_export.py

```diff
@@ -140,6 +141,16 @@ def stop_export(self):
             finally:
                 self._nbd_server_pid = None
 
+    def suspend_export(self):
+        if self._nbd_server_pid is not None:
+            try:
+                os.kill(self._nbd_server_pid, signal.SIGSTOP)
+            except Exception as e:
+                logging.warning("Error occurred when suspending"
+                                "nbd server: %s", str(e))
+            finally:
+                self._nbd_server_pid = None
```

Comment:
We cannot reset pid to None here, for we just suspend the process, not kill it.
Then stop_export can kill it and set pid to None

Reply:
it's better to add a continue function to send SIGCONT to resume the process, but as the current test needs not to cover this, it's OK for me to add this in future when it's required

Reply:
Thanks for your comment, remove operation reset pid to None. Maybe add resume process later, as the function itself won't be tested in this case.